pkg/README.md

```diff
@@ -8,13 +8,40 @@ this will hold all of the integrations / code that is to be used by the `cadctl`
 
 Use the `pagerduty.NewWithToken` to create the PagerDuty client, and use the functions it has
 
+#### Manutally test on an incident
+to do this, use the https://github.com/martindstone/pagerduty-cli tool.
+
+create the incident:
+```shell
+INCIDENT_TITLE= # the title you would want the incident to have
+SERVICE_ID= # the service under which the incident is created
+ESCALATION_POLICY_ID= # the escalation polict under which the incident is created (in case the default escalation policy will ping oncall
+pd incident:create --title=${INCIDENT_TITLE} --service_id=${SERVICE_ID} --escalation_policy_id=${ESCALATION_POLICY_ID}
+```
+the command will output the incident's id
+
+verify the incident has changed after running the `cadctl` change:
+```shell
+# for CLI data
+INCIDENT_ID= # the ID pulled from the previous command
+pd rest:get --endpoint /incidents/${INCIDENT_ID}
+# for the UI
+pd incident:open --ids ${INCIDENT_ID}
+```
+
+and from there you can resolve the incident:
+```shell
+INCIDENT_ID= # the ID pulled from the create command
+pd incident:resolve --ids ${INCIDENT_ID}
+```
+
 ## Development
 
 add a new endpoint to cadctl
 ```
 make bin/cobra
```

Comment:
this really works everywhere?

Reply:
Yes, I used it already 2 times

Reply:
@ramonbutter I didn't see what like this was referring to before.

anyways you can't run a makefile from a different folder

Reply:
this assumes you are in the root of the git folder, will change if this is unclear

Reply:
yea somehow I thought this makefile command was pointing to a path.
I tried it on my computer as well, but am receiving:
```
make bin/cobra                    
GOBIN=/home/ramon/git/configuration-anomaly-detection/bin go install github.com/spf13/cobra/cobra@v1.3.0
package github.com/spf13/cobra/cobra@v1.3.0: can only use path@version syntax with 'go get'
make: *** [Makefile:3: bin/cobra] Error 1

```

Reply:
what is your `go version`?

Reply:
go version go1.15.15 linux/amd64

Reply:
this doesn't reproduce on my machine :(

Reply:
nvm

Reply:
now it does. if you run the `make bin/cobra` target again and it should warn you